css/shortcode-ui.css

```diff
@@ -20,16 +30,16 @@
     text-align: center;
     width: 150px;
     height: 150px; }
-    .add-shortcode-list .shortcode-list-item .add-shortcode-list-item-icon {
+    .insert-shortcode-list .add-shortcode-list .shortcode-list-item .add-shortcode-list-item-icon {
       position: relative;
       height: 120px;
       font-size: 64px; }
-      .add-shortcode-list .shortcode-list-item .add-shortcode-list-item-icon .dashicons, .add-shortcode-list .shortcode-list-item .add-shortcode-list-item-icon img {
+      .insert-shortcode-list .add-shortcode-list .shortcode-list-item .add-shortcode-list-item-icon .dashicons,
+      .insert-shortcode-list .add-shortcode-list .shortcode-list-item .add-shortcode-list-item-icon img {
         position: absolute;
         top: 50%;
         left: 50%;
         -webkit-transform: translate(-50%, -50%);
-            -ms-transform: translate(-50%, -50%);
```

Comment:
I guess autoprefixer has decided it's no longer necessary? I think it's fine to go with that - if we need to be more explicit about which browsers we're targeting - we should update the autoprefixer task - but I'd tackle that separately.